Prox/Prox/PlaceDetails/PlaceDetailsDescriptionView.swift

```diff
@@ -130,6 +130,7 @@ class PlaceDetailsDescriptionView: UIView {
             expandableLabelHeightConstraint.isActive = true
             logoBottomConstraint.constant = 0
         }
-        layoutIfNeeded()
+
+        setNeedsLayout()
```

Comment:
it's needed only for when we expand and contract the views once displayed to apply the layout changes, not required during setup of the views.